src/commands/flutter-symbols/upload.ts

```diff
@@ -0,0 +1,444 @@
+// Disabling no-unused-variable / no-empty temporarily so certain private methods
+// can remain in while this feature is in progress
+// tslint:disable: no-unused-variable
+// tslint:disable: no-empty
+import fs from 'fs'
+
+import {Command} from 'clipanion'
+import yaml from 'js-yaml'
+import {
+  renderArgumentMissingError,
+  renderCommandInfo,
+  renderCommandSummary,
+  renderFailedUpload,
+  renderGeneralizedError,
+  renderGitWarning,
+  renderInvalidPubspecError,
+  renderInvalidSymbolsDir,
+  renderMissingAndroidMappingFile,
+  renderMissingDartSymbolsDir,
+  renderMissingPubspecError,
+  renderPubspecMissingVersionError,
+  renderRetriedUpload,
+  renderUpload,
+  UploadInfo,
+} from './renderer'
+
+import glob from 'glob'
+import asyncPool from 'tiny-async-pool'
+import {ApiKeyValidator, newApiKeyValidator} from '../../helpers/apikey'
+import {getRepositoryData, RepositoryData} from '../../helpers/git/format-git-sourcemaps-data'
+import {getMetricsLogger, MetricsLogger} from '../../helpers/metrics'
+import {MultipartValue, UploadStatus} from '../../helpers/upload'
+import {buildPath, DEFAULT_CONFIG_PATH, performSubCommand, resolveConfigFromFile} from '../../helpers/utils'
+import * as dsyms from '../dsyms/upload'
+import {newSimpleGit} from '../git-metadata/git'
+import {getArchInfoFromFilename, getFlutterRequestBuilder, uploadMultipartHelper} from './helpers'
+import {
+  DART_SYMBOL_FILE_NAME,
+  JVM_MAPPING_FILE_NAME,
+  MappingMetadata,
+  TYPE_DART_SYMBOLS,
+  TYPE_JVM_MAPPING,
+  VALUE_NAME_DART_MAPPING,
+  VALUE_NAME_JVM_MAPPING,
+} from './interfaces'
+
+export class UploadCommand extends Command {
+  public static usage = Command.Usage({
+    description: '',
+    details: `
+        `,
+    examples: [],
```

Comment:
Is there a reason to keep these empty for now? I believe they help for understanding the command.

Reply:
👍 Absolutely right, I'll add those in.